File: app/src/main/java/com/frankegan/verdant/welcome/WelcomeContract.java
package com.frankegan.verdant.welcome;

/**
 * Created by frankegan on 5/15/16.
 */
public interface WelcomeContract {
    interface View {

        void setWelcomeName(String accountName);

        void close();
    }

    interface UserActionsListener {

        void explore();

        void saveUser();
    }
}

File: app/src/main/java/com/frankegan/verdant/welcome/WelcomeContract.kt
package com.frankegan.verdant.welcome

/**
 * Created by frankegan on 5/15/16.
 */
interface WelcomeContract {
    interface View {

        fun setWelcomeName(accountName: String)

    }

    interface UserActionsListener {

        fun saveUser()
    }
}
